Magento2/Sniffs/Functions/DiscouragedFunctionSniff.php

```diff
@@ -238,5 +238,12 @@ class DiscouragedFunctionSniff extends ForbiddenFunctionsSniff
         '^fseek$' => 'Magento\Framework\Filesystem\DriverInterface::fileSeek()',
         '^feof$' => 'Magento\Framework\Filesystem\DriverInterface::endOfFile()',
         '^flock$' => 'Magento\Framework\Filesystem\DriverInterface::fileLock()',
+        '^date_sunrise$' => 'date_sun_info',
+        '^date_sunset$' => 'date_sun_info',
+        '^strptime$' => 'date_parse_from_format',
+        '^strftime$' => null,
```

Comment:
It would be great to add replacement recommendations for each of the deprecated functions. For example for this function:
```suggestion
        '^strftime$' => 'IntlDateFormatter with \Locale::getDefault()',
```

Reply:
Even though the replacement can be more complex than just a single function call, like in this case

```
setlocale(LC_ALL, 'nl_NL');
$formatter = new IntlDateFormatter(\Locale::getDefault(), IntlDateFormatter::FULL, IntlDateFormatter::FULL);
$formatter->setPattern('EEEE, LLLL d');
echo $formatter->format(new DateTime());
```
The replacement suggestion should at least help the developer with the research direction.